partials/re_verify.html

```diff
@@ -0,0 +1,51 @@
+<% extends "partials/base/base.html" %>
+
+<% block content %>
+<!--Header Area-->
+<tr style="page-break-before: always">
+  <td valign="top">
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{{t "You signed in to Firefox from [OS], [Location, Location, Location] Please confirm this sign-in to begin syncing with [OS]." }}}<% endblock %></p>
```

Comment:
edit: instead of `%(deviceName)s`, use `%(device)s` to be consistent with the suspicious location template.

There are no links here, no need for `{{{..}}}`. Please convert to `{{..}}`